docs/_releases/v0.5.16.md

```diff
@@ -12,13 +12,13 @@ tag: v0.5.16
 - [update] Filter UI @Chadha93 (#3005)
 - Add platform support to bash script @navendu-pottekkat (#2997)
 
-## Features
+**Features**
 - [mesheryctl] Improved UX for mesheryctl system check --pre @piyushsingariya (#3027)
 
-## 🧰 Maintenance
```

Comment:
Gents, a verbatim copy of the release notes to a new .yaml file with appropriate frontmatter in the Docs is all that is needed. No reformatting needs to be done.

Reply:
Copy that, but what about the ones that are released? So far I made changes only to ones that are released and published in docs. For future ones... IG we can fix the CI action and take care of that

Reply:
Well I'll reconsider then....

Reply:
Okay, affirmative captain 👀